Formula/newrelic-infra-agent.rb

```diff
@@ -42,8 +42,8 @@ def install
   end
 
   def post_install
-    (etc/"newrelic-infra").mkpath
-    (var/"log/newrelic-infra").mkpath
+    # (etc/"newrelic-infra").mkpath
+    # (var/"log/newrelic-infra").mkpath
```

Comment:
Why are we commenting this out?

Reply:
Good catch! This PR was done automatically by bump-formula tool.

I will push a commit to fix it. Thanks

Reply:
It would be a bit shocking that `bump-formula-pr` command would comment out any code.

Reply:
This is the cmd code, https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb